src/asm/charmap.c

```diff
@@ -91,6 +24,8 @@ uint32_t decode(uint32_t *state, uint32_t *codep, uint32_t byte)
 #include "asm/main.h"
 #include "asm/output.h"
 
+uint32_t decode(uint32_t *state, uint32_t *codep, uint32_t byte);
```

Comment:
I'd personally put this in a header file for `utf8decoder.h`, just to make it obvious where this function lives, but I'm just splitting hairs here 😛

Reply:
Done. I was just a bit lazy when I created that patch.